packages/Core/controlFlow.candy

```diff
@@ -1,7 +1,7 @@
 bool = use "..bool"
-equals = (use "..equality").equals
+[equals] = use "..equality"
 function = use "..function"
-isSymbol = (use "..symbol").is
+symbol = use "..symbol"
```

Comment:
I was confused why this didn’t cause code below to change. Turns out, the symbol module is never used. So I guess you can also remove all of this